azurelinuxagent/common/osutil/factory.py

```diff
@@ -84,28 +84,23 @@ def _get_osutil(distro_name, distro_code_name, distro_version, distro_full_name)
         return CoreOSUtil()
 
     if distro_name in ("suse", "sles", "opensuse"):
-        # pylint: disable=R1705
-        if distro_full_name == 'SUSE Linux Enterprise Server' \
-                and Version(distro_version) < Version('12') \
+        if distro_full_name == 'SUSE Linux Enterprise Server' and Version(distro_version) < Version('12') \
                 or distro_full_name == 'openSUSE' and Version(distro_version) < Version('13.2'):
             return SUSE11OSUtil()
-        else:
-            return SUSEOSUtil()
+
+        return SUSEOSUtil()
 
     if distro_name == "debian":
-        if "sid" in distro_version or Version(distro_version) > Version("7"):  # pylint: disable=R1705
+        if "sid" in distro_version or Version(distro_version) > Version("7"):
             return DebianOSModernUtil()
-        else:
-            return DebianOSBaseUtil()
-
-    # pylint: disable=R1714
-    if distro_name == "redhat" \
-            or distro_name == "centos" \
-            or distro_name == "oracle":
-        if Version(distro_version) < Version("7"):  # pylint: disable=R1705
+
+        return DebianOSBaseUtil()
+
+    if distro_name in ("redhat", "rhel", "centos", "oracle"):
```

Comment:
Adding "rhel" here is the primary change of the PR, everything else is just getting rid of linter warnings.

Reply:
Sounds about right.